vizro-core/src/vizro/models/_components/form/_user_input.py

```diff
@@ -44,13 +44,15 @@ def build(self):
                 dbc.Input(
                     id=self.id,
                     placeholder=self.placeholder,
+                    # Note that "range" and "hidden" are not included as possible `input_type` as our recommended
```

Comment:
Wouldn't the same be true for `"number"` also?

Actually I think we may well want to just fix `input_type="text"` here and not expose the field at all. Doesn't really matter for now since it's not public, but if you agree then let's just remove the field entirely.

Reply:
Yes, you are right! I think the only input types that are quite useful are `email` and `password`. This will most likely be used for authentication purposes though, where I think we currently recommend leveraging existing systems. 

So let me remove it for now and see how people might want to use it 👍